src/app/views/Publishes.svelte

```diff
@@ -43,28 +51,35 @@
 </script>
 
 <Subheading>Published Events</Subheading>
-<div class="grid grid-cols-4 justify-between gap-2 sm:grid-cols-5">
-  <Tile background>
-    <p class="text-lg sm:text-2xl">{recent.length}</p>
-    <span class="text-sm">{pluralize(recent.length, "Event")}</span>
-  </Tile>
-  <Tile background>
-    <p class="text-lg sm:text-2xl">{relays.size}</p>
-    <span class="text-sm">{pluralize(relays.size, "Relay")}</span>
-  </Tile>
-  <Tile background lass="hidden sm:block">
-    <p class="text-lg sm:text-2xl">{pending.length}</p>
-    <span class="text-sm">Pending</span>
-  </Tile>
-  <Tile background>
-    <p class="text-lg sm:text-2xl">{success.length}</p>
-    <span class="text-sm">Succeeded</span>
-  </Tile>
-  <Tile background>
-    <p class="text-lg sm:text-2xl">{recent.length - pending.length - success.length}</p>
-    <span class="text-sm">Failed</span>
-  </Tile>
-</div>
-{#each sortBy(t => -t.event.created_at, recent) as thunk (thunk.event.id)}
-  <PublishCard {thunk} />
-{/each}
+<Tabs {tabs} {activeTab} setActiveTab={tab => (activeTab = tab)} />
+{#if activeTab === "events"}
+  <div class="grid grid-cols-4 justify-between gap-2 sm:grid-cols-5">
+    <Tile background>
+      <p class="text-lg sm:text-2xl">{recent.length}</p>
+      <span class="text-sm">{pluralize(recent.length, "Event")}</span>
+    </Tile>
+    <Tile background>
+      <p class="text-lg sm:text-2xl">{relays.size}</p>
+      <span class="text-sm">{pluralize(relays.size, "Relay")}</span>
+    </Tile>
+    <Tile background lass="hidden sm:block">
+      <p class="text-lg sm:text-2xl">{pending.length}</p>
+      <span class="text-sm">Pending</span>
+    </Tile>
+    <Tile background>
+      <p class="text-lg sm:text-2xl">{success.length}</p>
+      <span class="text-sm">Succeeded</span>
+    </Tile>
+    <Tile background>
+      <p class="text-lg sm:text-2xl">{recent.length - pending.length - success.length}</p>
+      <span class="text-sm">Failed</span>
+    </Tile>
+  </div>
+  {#each sortBy(t => -t.event.created_at, recent) as thunk (thunk.event.id)}
+    <PublishCard {thunk} />
+  {/each}
```

Comment:
Can we move this into a separate component?

Reply:
I avoid splitting the views atm, I don't know where to place or how to name those partials component, the list is gigantic. 
Moving to Sveltekit router in a future version would give us more structure and options for organising those components.